src/sentry/replays/blueprints/api.md

```diff
@@ -0,0 +1,366 @@
+# Replays API
+
+Host: https://sentry.io/api/0
+
+**Authors.**
+
+@cmanallen
+@joshferge
+
+**How to read this document.**
+
+This document is structured by resource with each resource having actions that can be performed against it. Every action that either accepts a request or returns a response WILL document the full interchange format. Clients may opt to restrict response data or provide a subset of the request data. The API may or may not accept partial payloads.
+
+## Replays [/organizations/<organization_slug>/replays/]
+
+- Parameters
+
+  - field (optional, string)
+  - environment (optional, string)
+  - project (optional, string)
+  - sort (optional, string)
+    Default: -startedAt
+    Members: + projectId + -projectId + startedAt + -startedAt + finishedAt + -finishedAt + duration + -duration + countErrors + -countErrors
+  - statsPeriod (optional, string) - A positive integer suffixed with a unit type.
+    Default: 7d
+    Members: + s + m + h + d + w
+  - start (optional, string) - ISO 8601 format (`YYYY-MM-DDTHH:mm:ss.sssZ`)
+  - end (optional, string) - ISO 8601 format. Required if `start` is set.
+  - limit (optional, number)
+    Default: 10
+  - offset (optional, number)
+    Default: 0
+  - query (optional, string) - Search query with space-separated field/value pairs. ie: `?query=count_errors:>2 AND duration:<1h`.
+
+    Some fields in the API response have their own dedicated parameters, or are otherwide not supported in the `query` param. They are:
+
+    | Response Field      | Parameter       |
+    | ------------------- | --------------- |
+    | `environment`       | `?environment=` |
+    | `project_id`        | `?project=`     |
+    | `started_at`        | `?start=`       |
+    | `finished_at`       | `?end=`         |
+    | `user.display_name` | -               |
+
+    You can use the following aliases to query for fields that are plural in the API response:
+
+    | Response Field | Search Alias         |
+    | -------------- | -------------------- |
+    | `error_ids`    | `error_id`           |
+    | `releases`     | `release`            |
+    | `trace_ids`    | `trace` & `trace_id` |
+    | `urls`         | `url`                |
+
+    Additionally, you can filter by these hidden fields.
+
+    | Field        | Type          | Description                          |
+    | ------------ | ------------- | ------------------------------------ |
+    | action       | enum[string]  | Choices: click                       |
+    | tag          | string        | Valid HTML5 tag name.                |
+    | id           | string        | The ID of an HTML element.           |
+    | classes      | array[string] | An array of HTML element classes.    |
+    | role         | string        | The role of an HTML element.         |
+    | aria-label   | string        | The aria-label of an HTML element.   |
```

Comment:
I wonder if we just name this `label` to avoid the inconsistency of using `-` vs `_` (our search is snake, but we'd want to match the attribute name).